bundle/manifests/node-maintenance-operator.clusterserviceversion.yaml

```diff
@@ -274,17 +274,17 @@ spec:
   - supported: false
     type: AllNamespaces
   keywords:
-  - KubeVirt
+  - medik8s
   - Virtualization
```

Comment:
you can remove Virtualization

Reply:
Should I add `Dragonfly` as the team's name?